models/webhook.go

```diff
@@ -110,3 +80,26 @@ func (f WebhookEventFilters) MergeWithDefaults() WebhookEventFilters {
 	}
 	return defaultFilters
 }
+
+type WebhookCreate struct {
+	OrganizationId    string
+	PartnerId         null.String
+	EventType         WebhookEventType
```

Comment:
as discussed orally: the PR looks globally good to me, the one thing I'm unsure about is I'd rather see a mapping 1 registered webhook <-> list of event types